components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -166,8 +165,8 @@ void JsonRpcService::AddObserver(
 void JsonRpcService::Request(const std::string& json_payload,
                              bool auto_retry_on_network_change,
                              RequestCallback callback) {
-  RequestInternal(json_payload, auto_retry_on_network_change, network_url_,
-                  std::move(callback));
+  RequestInternal(json_payload, auto_retry_on_network_change,
+                  network_urls_[mojom::CoinType::ETH], std::move(callback));
```

Comment:
I think `Request` should take in CoinType and use `network_urls_[coin_type]` when calling `RequestInternal`.
`Request` is shared with solana and filecoin requests too.

Reply:
fixed in https://github.com/brave/brave-core/pull/12461/commits/cea463abc30fc02b69d035171fd3525822495657